cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -3618,7 +3618,11 @@ mod tests {
                 r#"permit(principal is User in User, action, resource);"#,
                 ExpectedErrorMessageBuilder::error(
                     "expected an entity uid or matching template slot, found name `User`",
-                ).exactly_one_underline("User").build(),
+                )
+                .help(
+                    "try using `is` to test for an entity type"
```

Comment:
The help text in this case & the next is not too useful since the policy already uses `is`. Maybe adjust the casing to only add this help text when `_ is _ in _` isn't used?